hack/ci/windows.ps1

```diff
@@ -960,6 +960,9 @@ Catch [Exception] {
     Throw $_
 }
 Finally {
+    # Preserve the LastExitCode of the tests
+    $tmpLastExitCode = $LastExitCode
```

Comment:
@StefanScherer this was just an attempt to see if this would help with the RS1 builds sometimes not exiting; not sure if this would help or not (happy to drop this commit if it doesn't make sense)

Reply:
Moved this to a separate PR https://github.com/moby/moby/pull/39854